assets/scss/templates/_reportes.scss

```diff
@@ -79,7 +79,7 @@
 }
 
 /* Section KPI */
-.container-kpi {
+.container-reports {
```

Comment:
It is a breaking change, what do you think about keeping the old class (with a `// TODO` comment), in order to keep compatibility?

This kind of practices is not required right now, but we should at least think about it in order to learn to think in that way.

Reply:
Lo cambie por que es un container general para "reportes" no solo de kpi. Pero como quieran, se cambia